src/admin/reducers/search.js

```diff
@@ -1,4 +1,5 @@
 const initialState = {
+  auth: true,
```

Comment:
It looks like `amo/reducers/search.js` is identical except for this line. Is this needed though? Could there be one reducer in `core` that handles both?